apps/examples/sensorbd_demo/examples/i2c_mpu9250.c

```diff
@@ -197,7 +194,7 @@ int mpu9250_main(int argc, char *argv[])
 			break;
 		}
 
-		printf("ACC:      %-10d%-10d%-10d\n", (int16_t)data[0], (int16_t)data[1], (int16_t)data[2]);
+		printf("ACC:      %-10d%-10d%-10d\n", (int16_t) data[0], (int16_t) data[1], (int16_t) data[2]);
```

Comment:
As far as I know, we do not need to put a space after type casting. Why do you put a space here? Very confused.